--- a/Ghidra/Features/Decompiler/src/decompile/cpp/userop.cc	2026-05-04 00:00:00.000000000 -0700
+++ b/Ghidra/Features/Decompiler/src/decompile/cpp/userop.cc	2026-05-04 00:00:00.000000000 -0700
@@ -595,6 +595,9 @@
     registerOp(op);
   } catch(LowlevelError &err) {
     delete op;
+    const string unknownUseropPrefix = "Unknown userop name in <callotherfixup>: ";
+    if (err.explain.rfind(unknownUseropPrefix, 0) == 0)
+      return;
     throw err;
   }
 }
